pimd/subdir.am

```diff
@@ -74,6 +74,15 @@ nodist_pimd_pimd_SOURCES = \
 	yang/frr-igmp.yang.c \
 	# end
 
+pimd_pim6d_SOURCES = \
+	$(pim_common) \
+	pimd/pim6_main.c \
+	pimd/pim6_cmd.c \
```

Comment:
Add pimd/pim6_cmd.h in this section noinst_HEADERS += \

Reply:
Added